internal/postgres/config.go

```diff
@@ -303,27 +303,27 @@ chmod +x /tmp/pg_rewind_tde.sh
 
 		// Log the effective user ID and all the group IDs.
 		`echo Initializing ...`,
-		`results 'uid' "$(id -u)" 'gid' "$(id -G)"`,
+		`results 'uid' "$(id -u ||:)" 'gid' "$(id -G ||:)"`,
 
 		// Abort when the PostgreSQL version installed in the image does not
 		// match the cluster spec.
-		`results 'postgres path' "$(command -v postgres)"`,
-		`results 'postgres version' "${postgres_version:=$(postgres --version)}"`,
+		`results 'postgres path' "$(command -v postgres ||:)"`,
```

Comment:
Should we be checking earlier in this script that we have postgres on this container?

Reply:
Most things above here are function declarations. The only calls ahead of this are `printf` and `id`, so seems early enough to me.